salt/modules/cmdmod.py

```diff
@@ -52,8 +52,10 @@
 
 if salt.utils.platform.is_windows():
     from salt.utils.win_runas import runas as win_runas
+    from salt.utils.win_functions import escape_argument as _cmd_quote
     HAS_WIN_RUNAS = True
 else:
+    from salt.ext.six.moves import shlex_quote as _cmd_quote
```

Comment:
The `oxygen` branch added the cmd_quote import on line 45 above, but this is a better fix for this, so please remove line 45.

Reply:
👍